### PR TITLE
Add workflows

### PR DESCRIPTION
**Pull request by Chandraprakash-Darji**
## Related Issue

Add the CI for the deploy, lint, and prettier fix to bug_report, feature_request, and add-event

Closes: `No issue`

### Describe the changes you've made

created files for deploy and lint 

## Type of change

What sort of change have you made:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update.
- [ ] This change requires a documentation update

## How Has This Been Tested?

- [x] My code follows the guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented on my code, particularly wherever it was hard to understand.
- [ ] I have made corresponding changes to the documentation.
- [x] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] Any dependent changes have been merged and published in downstream modules.

## Screenshots (if applicable)

|      Original       |      Updated       |
| :-----------------: | :----------------: |
| No| No |

## Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/WebXDAO/.github/blob/main/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        with:
+          cmd: install # will run `yarn install` command
+      # Building app
+      - uses: borales/actions-yarn@v3.0.0
+        with:
+          cmd: build # will run `yarn build` command
+
+  deploy:
+    needs: build
+    if: github.repository_owner == 'ax-studios' && ((github.event.pull_request.head.repo.full_name == github.repository) || (github.event_name == 'push'))
```
Comment (mkubdev) [inline .github/ISSUE_TEMPLATE/deploy.yml]:
```suggestion
    if: github.repository_owner == 'webxdao' && ((github.event.pull_request.head.repo.full_name == github.repository) || (github.event_name == 'push'))
```

**Review by mkubdev — APPROVED** [review]
There is a repo_owner to update, but LGTM 🎄

**Comment by Chandraprakash-Darji** [comment]
image
What checks are failed?

File: .github/ISSUE_TEMPLATE/deploy.yml (hunk 1 above)
Comment (Chandraprakash-Darji) [reply .github/ISSUE_TEMPLATE/deploy.yml]: commited changes

**Comment by mkubdev** [comment]
> image What checks are failed?

Failed due to the need to authorize the PR build. I'm the only one who can validate builds, that's kind of annoying and i need to find a way to solve  this. 

> **The latest updates on your projects**. Learn more about [Vercel for Git ↗︎](https://vercel.link/github-learn-more)
> 
> Name	Status	Preview	Updated
> **dev-conf**	✅ Ready ([Inspect](https://vercel.com/web-xdao/dev-conf/4d1yVu6iusoMvTnV1dTFUwKsaQf8))	[Visit Preview](https://dev-conf-git-fork-chandraprakash-darji-add-workflows-web-xdao.vercel.app)	Dec 5, 2022 at 10:27AM (UTC)

But it look after authorization, the PR building correctly.

**Pull request merged**